### ISSUE TITLE
Ship Humanizer 2.1

### ISSUE THREAD

**Issue by onovotny** [Humanizr/Humanizer#567]
Issue to track shipping Humanizer 2.1.

In addition to the bug fixes since `2.0.1`, this version will support `netstandard`. As such, it will also require a minimum NuGet version of `2.12` to install correctly.

@hazzik @aloisdg @mexx  Please add items to this list that need to be complete before we ship.

- [x] Merge in `netstandard` branch
- [x] Ensure assembly versions are updated #535
- [x] Modified PascalCaseWordPartsRegex to accept more "word characters" #564 
- [x] Handling of null, blank and untrimmed collection items #566 
 - [x] Resolve the issue with overloads instead of optional arguments

**Comment by aloisdg** [comment]
I am working to port [Numbro](https://github.com/aloisdg/Numbro) to Humanizer. The related issue is  #558. The PR should be ready as soon as  #568 is fixed and I [achieve to build](https://gitter.im/Humanizr/Humanizer?at=5770718e1ac8bd1a4d613cef) 😄 

Also:

-  I want to support Esperanto DateTime, but the code doesn't exist yet.
- The support for the french "avant-hier/" and "après-demain" will be postpone.
- There are old codes inside NumberToWords. I would like to move it C#6 (string interpolation, etc.). It could be postpone since it would be plain refactoring.

One item is sure to be here:

- [ ] Add esperanto support to NumberToWords

**Comment by hazzik** [comment]
I would like to nominate following:

- [ ] Modified PascalCaseWordPartsRegex to accept more "word characters" #564 
- [ ] Handling of null, blank and untrimmed collection items #566 
 - [ ] Resolve the issue with overloads instead of optional arguments

**Comment by aloisdg** [comment]
@onovotny Can you update the milestone in each issue/PR?

**Comment by aloisdg** [comment]
I would like to propose #518 by @i3arnon (PR: #519)

**Comment by hazzik** [comment]
What if we just let the GitVersion decide what version is going next?

**Comment by onovotny** [comment]
@hazzik GitVersion is just a tool...it does whatever we want it to be. In general, here's a good take on it:
https://docs.microsoft.com/en-us/dotnet/articles/core/versions/index#semantic-versioning

By that logic, it feels like a 2.1 for this release.

**Comment by hazzik** [comment]
I know that this shall be a 2.1. But what I'm trying to say: we do `+semver:` hints on commits, and just release whatever version is calculated by GitVersion.

**Comment by hazzik** [comment]
Hey-hey. I've merged #566. :ship: the 2.1.

**Comment by onovotny** [comment]
I generally tag the final version and that will automatically set it in GitVersion

**Comment by onovotny** [comment]
Note that Humanizer is completely broken on dotnet CLI due to this: https://github.com/dotnet/cli/issues/3396

Try it today with the MyGet/CI feed; you cannot restore it into an xproj.

**Comment by onovotny** [comment]
I may have a workaround for this that I'm testing -- a `Humanizer.xproj` package that has all of the files and resources in the same package (like Humanizer 1.x did before the split). This should work but I'll verify. I'll nuke the package once the cli bug is fixed or things transition to msbuild in the fall.

**Comment by onovotny** [comment]
Confirmed, the `Humanizer.xproj` works. Now just need docs and release notes and we're good to go.

**Comment by onovotny** [comment]
2.1 has been pushed to NuGet. Should be visible soon.
